### ISSUE TITLE
Improve the quality of migrated Fluent files

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#8045]
## Description

before we start porting legacy .lang files over to .ftl, we should make sure we try and adhere to a defined level of quality for those files. This could be achieved by a combination of improved automation and / or documentation.

**Comment by alexgibson** [comment]
- Order of strings in fluent file should follow order in template.
- Avoid superfluous comments from .lang files that are no longer needed.
- Quality of auto-generated IDs?
- URLs should point to a resolving page.

**Comment by alexgibson** [comment]
@Pike do you feel like the work in https://github.com/mozilla/bedrock/pull/8087 was enough to cover the main points above? Can this issue be closed?

**Comment by alexgibson** [comment]
I'm going to close this based on the guidelines that were added to https://bedrock.readthedocs.io/en/latest/fluent-conversion.html

Feel free to re-open if there's still something missing in our process / documentation.
